### PR TITLE
Cleanup Javadoc etc.

### PR DESCRIPTION
**Pull request by jim-krueger**

### CODE REVIEW

File: NOTICE.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Notices for Eclipse Project for Stable Jakarta EE APIs
+# Notices for  Jakarta Stable APIs
 
 This content is produced and maintained by the Eclipse Project for Stable
```
Comment (kwsutter) [inline NOTICE.md]: Update project name here also.  I didn't look very closely, did we also miss these on the management repo?

File: NOTICE.md [hunk 2]
```diff
@@ -7,7 +7,7 @@ Jakarta EE APIs project.
 
 ## Trademarks
 
-Eclipse Project for Stable Jakarta EE APIs is a trademark of the Eclipse
+Eclipse Project for Jakarta Stable APIs is a trademark of the Eclipse
```
Comment (kwsutter) [inline NOTICE.md]: Just "Jakarta Stable APIs" is a trademark...  Drop the Eclipse Project for...

File: api/pom.xml [hunk 3]
```diff
@@ -20,14 +20,14 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
     <modelVersion>4.0.0</modelVersion>
     <parent>
-        <groupId>org.eclipse.ee4j.xml.registry-api</groupId>
-        <artifactId>xml.registry-api-parent</artifactId>
-        <version>1.0.9-SNAPSHOT</version>
+        <groupId>jakarta.xml.registries-api</groupId>
```
Comment (kwsutter) [inline api/pom.xml]: Nope, don't change these coordinates.  It's just the name of the project, not the coordinates.  That would mess up existing users.

File: api/pom.xml [hunk 4]
```diff
@@ -20,14 +20,14 @@
...
+        <artifactId>xml.registries-api-parent</artifactId>
+        <version>1.0.10-SNAPSHOT</version>
     </parent>
 
-    <groupId>jakarta.xml.registry</groupId>
-    <artifactId>jakarta.xml.registry-api</artifactId>
-    <version>1.0.9-SNAPSHOT</version>
+    <groupId>jakarta.xml.registries</groupId>
```
Comment (kwsutter) [inline api/pom.xml]: Same here, don't change.

File: api/pom.xml [hunk 5]
```diff
@@ -160,7 +160,7 @@
                         <Implementation-Version>${spec.implementation.version}</Implementation-Version>
                         <Specification-Version>${spec.specification.version}</Specification-Version>
                         <Bundle-Description>
-                            Java(TM) JAXR ${spec.version} API Design Specification
+                            Jakarta XML Registries ${spec.version} API Design Specification
```
Comment (kwsutter) [inline api/pom.xml]: Put the (TM) after Jakarta.

File: api/pom.xml [hunk 6]
```diff
@@ -236,13 +236,21 @@
...
                 <version>2.10.3</version>
                 <configuration>
-		        <docfilessubdirs>true</docfilessubdirs>
+                    <additionalparam>-Xdoclint:none</additionalparam>
+                    <bottom>
+                      <![CDATA[
+                      <p align="left">&#169; Copyright 2019 Eclipse Foundation.<br>Use is subject to
+                      <a href="{@docRoot}/doc-files/EFSL.html" target="_top">license terms</a>.]]>
+                    </bottom>
+		            <docfilessubdirs>true</docfilessubdirs>
```
Comment (kwsutter) [inline api/pom.xml]: Can you fix up the indentation?

File: pom.xml [hunk 7]
```diff
@@ -26,10 +26,10 @@
         <version>1.0.5</version>
     </parent>
 
-    <groupId>org.eclipse.ee4j.xml.registry-api</groupId>
-    <artifactId>xml.registry-api-parent</artifactId>
+    <groupId>jakarta.xml.registries-api</groupId>
```
Comment (kwsutter) [inline pom.xml]: Use "jakarta" but leave the "registry" in the GAV

File: pom.xml [hunk 8]
```diff
@@ -26,10 +26,10 @@
...
+    <artifactId>xml.registries-api-parent</artifactId>
```
Comment (kwsutter) [inline pom.xml]: Leave "registry"

File: api/pom.xml [hunk 9]
```diff
@@ -40,7 +40,7 @@
         <findbugs.exclude>exclude.xml</findbugs.exclude>
         <findbugs.threshold>Low</findbugs.threshold>
     </properties>
-    <name>jaxr-api</name>
+    <name>${extension.name} API</name>
```
Comment (kwsutter) [inline api/pom.xml]: What is this "name" referring to?  How is it used?  If it's for the name of the repo, which it looks like, then this needs to stay as "jaxr-api".

File: pom.xml [hunk 10]
```diff
@@ -26,10 +26,10 @@
...
     <packaging>pom</packaging>
-    <version>1.0.9-SNAPSHOT</version>
+    <version>1.0.10-SNAPSHOT</version>
     <name>jaxr-api</name>
     <description>JAX R API: Eclipse Project for Stable EE4J APIs</description>
```
Comment (kwsutter) [inline pom.xml]: Description needs updating.

File: spec/pom.xml [hunk 11]
```diff
@@ -18,12 +18,12 @@
 -->
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
     <parent>
-        <groupId>org.eclipse.ee4j.xml.registry-api</groupId>
-        <artifactId>xml.registry-api-parent</artifactId>
-        <version>1.0.9-SNAPSHOT</version>
+        <groupId>jakarta.xml.registries-api</groupId>
+        <artifactId>xml.registries-api-parent</artifactId>
```
Comment (kwsutter) [inline spec/pom.xml]: Leave artifactId, but clean up groupId.

File: spec/pom.xml [hunk 12]
```diff
@@ -18,12 +18,12 @@
...
+        <version>1.0.10-SNAPSHOT</version>
     </parent>
     <modelVersion>4.0.0</modelVersion>
-    <groupId>jakarta.xml.registry</groupId>
+    <groupId>jakarta.xml.registries</groupId>
```
Comment (kwsutter) [inline spec/pom.xml]: Nope.

File: NOTICE.md (hunk 1 above)
Comment (jim-krueger) [reply NOTICE.md]: Yes... I'll fix that too.

File: NOTICE.md (hunk 2 above)
Comment (jim-krueger) [reply NOTICE.md]: Done

File: api/pom.xml (hunk 5 above)
Comment (jim-krueger) [reply api/pom.xml]: Done

File: pom.xml (hunk 7 above)
Comment (jim-krueger) [reply pom.xml]: Done

File: pom.xml (hunk 8 above)
Comment (jim-krueger) [reply pom.xml]: Done

File: api/pom.xml (hunk 9 above)
Comment (jim-krueger) [reply api/pom.xml]: I just copied what was done in management-api. See line 34 above this:

<extension.name>javax.xml.registry</extension.name>

File: pom.xml (hunk 10 above)
Comment (jim-krueger) [reply pom.xml]: <description>Jakarta XML Registries API: Eclipse Project for Stable EE4J APIs</description>

File: spec/pom.xml (hunk 12 above)
Comment (jim-krueger) [reply spec/pom.xml]: Done

File: spec/pom.xml (hunk 11 above)
Comment (jim-krueger) [reply spec/pom.xml]: Done

File: api/pom.xml (hunk 6 above)
Comment (jim-krueger) [reply api/pom.xml]: Done

File: api/pom.xml (hunk 9 above)
Comment (kwsutter) [reply api/pom.xml]: Okay.  It doesn't look like it's being used anyway...

**Pull request merged**